include/miniocpp/response.h

```diff
@@ -43,6 +43,10 @@ struct Response {
   std::string bucket_name;
   std::string object_name;
 
+ private:
+  error::Error err_;
+
+ public:
```

Comment:
Is it a good idea to group `private` and `public` fields and methods together?

Reply:
I think this is just a preference.

I personally don't care whether the members are on the top or bottom, but because in C++ every member contributes to the layout of the struct/class I like them grouped. In ideal case there should be an accessor so members are private and accessors are used to access them, but I don't want to change this at the moment - too much work and it would not be API compatible.